PICMI_Python/particles.py

```diff
@@ -27,8 +29,16 @@ class PICMI_Species(_ClassWithInit):
       - particle_shape: Particle shape used for deposition and gather ; if None, the default from the `Simulation` object will be used. Possible values are 'NGP', 'linear', 'quadratic', 'cubic'
     """
 
+    methods_list = ['Boris' , 'Vay', 'Higuera-Cary', 'Li', 'free-streaming', 'LLRK4']
+
     def __init__(self, particle_type=None, name=None, charge_state=None, charge=None, mass=None,
-                 initial_distribution=None, particle_shape=None, density_scale=None, **kw):
+                 initial_distribution=None, particle_shape=None, density_scale=None, method=None, **kw):
+
+
+        assert method is None or method in PICMI_Species.methods_list or method.startswith('other:'), \
```

Comment:
That convention looks good to me and is consistent with openPMD :+1:

Reply:
I don't think that these changes should be here. This is unrelated to the input file and these changes are already in the repo.

Reply:
I don't think that these changes should be here. This is unrelated to the input file and these changes are already in the repo.

Reply:
Oh, I didn't see that. Is this PR outdated and needs to merge `master`?

Reply:
I'm not sure why this diff is showing up here. Perhaps merge in master would help.

Reply:
yes, or rebasing the PR